src/comet/driver/light/__init__.py

```diff
@@ -0,0 +1 @@
+from .photonic import Photonic
```

Comment:
Convention for drivers is `<manufactor>/<model>.py` with class `<Model>`, like `keithley/k2410.py` providing class `K2410`. So `photonic/f3000.py` with a class `F3000` might work.

Reply:
See [f1a2412](https://github.com/hephy-dd/comet/pull/75/commits/f1a2412c303e672faf3adb4bb95b074ae3e8c6d1)